turingschool/savile
# Cut v1.2 release

#### What does this PR do?

Creates version 1.2 (which includes the `s-content` class!)

#### Where should the reviewer start?

All I did was change two files; this PR only has changes from the instructions in contributing.md.

#### How should this be manually tested?

If you want to run locally and visit http://localhost:4000/css/v1/1.2.css, you can cmd+F and see the `.s-content` class?

#### What are the relevant tickets?

https://3.basecamp.com/3494409/buckets/19192671/todos/3662357570

#### Any other deploy steps?

After merged I will follow step 5 in docs:
>Once merged make a new release in Github by going to https://github.com/turingschool/savile/releases and click 'Draft a new release'. Tag version (e.g v1.3),target (main) and release title (e.g 1.3), and what new features are included
